### DISCUSSION TITLE
Documentation help?

### DISCUSSION THREAD

**Discussion by dei-layborer** [ebullient/obsidian-slides-extended#16] [General]
The README says that documentation is borked ... is that just a matter of it not existing/not being updated for your fork, or is there some other issue?

I'd be glad to help on this if I can.

**Comment by ebullient** [comment]
I am not running hugo to make GH Pages. I have a loose plan for converting the shortcodes, I just have to get there.

In the meanwhile, the docs are just markdown, and outside of rendering, should be readable.
